README.md

```diff
@@ -114,12 +114,18 @@ To install Devika, follow these steps:
    ```
    python devika.py
    ```
-6. Compile and run the UI server:
+6. Compile and run the UI server:</br>
+   Install the Svelte package required
+   use command `npm install @sveltejs/vite-plugin-svelte` to install it
```

Comment:
`@sveltejs/vite-plugin-svelte` is already present in ui/package.json
`bun install` should install it. Am I missing something?

Reply:
It didn’t installed for me i used pip install -r requirements file

Reply:
You'd have to do `bun install` or `npm install`
`@sveltejs/vite-plugin-svelte` is a package in npm not pip

Reply:
But on Readme it was not mentioned now maybe its updated